Gemfile

```diff
@@ -2,7 +2,9 @@
 # If you modify this file manually all comments and formatting will be lost.
 
 source "https://rubygems.org"
-gem "logstash-core", "3.0.0.dev", :path => "."
+gem "logstash-core", "3.0.0.dev", :path => "./logstash-core"
+# gem "logstash-core-event", "3.0.0.dev", :path => "./logstash-core-event"
```

Comment:
as discussed, we will keep the current Ruby implementation as default for now. I don't have a problem leaving the Java one in comment since it allows switching in dev easily.